Code Examples/Andrii Stepaniuk/TestDataProvider.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Collections;
+
+namespace Stepaniuk.Homework.Tests
+{
+    class TestDataProvider : IEnumerable
+    {
+        public IEnumerator GetEnumerator()
+        {
+            yield return new string[] { "1", "John Smith", "admin@mail.com", "London", "Male" };
+            yield return new string[] { "2", "Jane Smith", "jane.smith@mail.com", "Tokio", "Female" };
```

Comment:
I think we should use some Indian names

Reply:
^ Totally agree with @mehhrunes (no issue there)

Reply:
I think the code should be changed to Indian too.

Reply:
@DR3AMcatcher , that would be perfect

Reply:
How about Chinese?